### PR TITLE
Added logging endpoint 

### LINKED ISSUE #23: Logging Endpoint
Drops our logs into some database.

### PR DESCRIPTION
**Pull request by RachitMalik12**
Added the following REST endpoints: 
- persistLogs: To persist logs to MongoDB
- getLogs: Get all logs
- getLog: Get a log with a particular id. 

Spec (see logging section of design doc): https://docs.google.com/document/d/1qDAzr3JpVODwLqw1Nyd_S_a-EBeuwF7QNtGX1_SrKHk/edit

### CODE REVIEW

File: api/src/models/logs.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import mongoose from 'mongoose';
+import { Guid } from "guid-typescript";
+
+const LogSchema = new mongoose.Schema({
+  logid: {
```
Comment (alanbzhang) [inline api/src/models/logs.ts]: change to logId

File: api/src/routes/logging.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import express, { Router, Request, Response, NextFunction } from 'express';
+const router: Router = express.Router();
+
+import validator from 'validator';
+
+const LogsPersistence = require('../models/logs');
```
Comment (alanbzhang) [inline api/src/routes/logging.ts]: change to something more inline with the other routes (e.g Log or PersistentLog)

File: api/src/app.ts [hunk 3]
```diff
@@ -19,6 +19,7 @@ app.use(bodyParser.json()); // parse JSON
 app.use('/api/account', require('./middleware/verify'), require('./routes/account'));
 app.use('/api/service', require('./middleware/verify'), require('./routes/service'));
 app.use('/api/request', require('./routes/request'));
+app.use('/api/logging', require('./middleware/verify'), require('./routes/logging'));
```
Comment (alanbzhang) [inline api/src/app.ts]: I don't think the verification middleware is necessary in this context

File: api/src/models/logs.ts (hunk 1 above)
Comment (RachitMalik12) [reply api/src/models/logs.ts]: ✅

File: api/src/routes/logging.ts (hunk 2 above)
Comment (RachitMalik12) [reply api/src/routes/logging.ts]: Will change to PersistentLog

File: api/src/app.ts (hunk 3 above)
Comment (RachitMalik12) [reply api/src/app.ts]: Removed.. ✅

**Pull request merged**